File: src/main/java/dev/choimory/member/api/member/v1/command/repository/MemberCommandRepository.java
package dev.choimory.member.api.member.v1.command.repository;

import dev.choimory.member.api.member.v1.command.domain.entity.MemberEntity;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

/**
 * 회원 command RDB 접근 Repository입니다.
 */
@Repository
public interface MemberCommandRepository extends JpaRepository<MemberEntity, String> {}

File: src/main/kotlin/dev/choimory/member/api/member/v1/command/repository/MemberCommandRepository.kt
package dev.choimory.member.api.member.v1.command.repository

import dev.choimory.member.api.member.v1.command.domain.entity.MemberEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface MemberCommandRepository : JpaRepository<MemberEntity, String>
